PetaPoco.Tests.Integration/Databases/OracleTests/OracleQueryTests.cs

```diff
@@ -26,8 +128,8 @@ public override void QueryMultiple_ForSingleResultsSetWithSinglePoco_ShouldRetur
             var pdName = pd.Columns.Values.First(c => c.PropertyInfo.Name == "Name").ColumnName;
 
             var sql = $@"SELECT *
-                         FROM {DB.Provider.EscapeTableName(pd.TableInfo.TableName)}
-                         WHERE {DB.Provider.EscapeSqlIdentifier(pdName)} LIKE @0 || '%';";
+    FROM {DB.Provider.EscapeTableName(pd.TableInfo.TableName)}
+    WHERE {DB.Provider.EscapeSqlIdentifier(pdName)} LIKE @0 || '%'";
 
```

Comment:
Re formatting, check whitespace diffing setting; looks like it was disabled, causing the string lines to lose their indentation.

Reply:
If you format verbatim strings to look pretty in the ide, you're
introducing additional characters in memory. I dedented those string to
give me the format I would use when writing scripts. It takes some time to
get used to, but is a much better developer experience.

Reply:
Completely agree on this; probably should have looked closer, as I wasn't thinking about it being a verbatim string vs concatenated interpolated string.